### DISCUSSION TITLE
Lab Assignment #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#137] [Assignments]
Paste your GitHub pages URL for `Week_05` as a comment below to submit your assignment.

e.g. https://akochaphum.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Comment by avaitt** [comment]
https://avaitt.github.io/21S-ASIAAM-191A-Assignments/Week_05/

**Reply by albertkun** [reply]
@avaitt haha, that survey looks pretty familiar! 😃   I wonder if you can see what marker I added.

**Comment by rachan2023** [comment]
https://rachan2023.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@rachan2023 wow! nice data fields!! but i can't take the survey! :(

![image](https://user-images.githubusercontent.com/8574425/116800027-37ae4300-aab2-11eb-8986-016cf44a07d4.png)

**Reply by rachan2023** [reply]
I think I fixed it now! Sorry about that!

**Reply by albertkun** [reply]
@rachan2023 no problem! see if you can find my point!  😄

**Comment by ghost** [comment]
https://ktgreenler.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@ktgreenler nice!!  good questions! i added a data point to your survey!

**Comment by latishasumardy** [comment]
https://latishasumardy.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@latishasumardy unable to see survey, please fix permissions when you get a chance.

**Comment by jason-muljadi** [comment]
https://jason-muljadi.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@jason-muljadi nice! added a little marker about my sentimental place.

**Comment by alizezila** [comment]
https://alizezila.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@alizezila dropped off a marker! Great theme!

**Comment by nnhimichelle** [comment]
https://nnhimichelle.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@nnhimichelle Uh oh, I added a marker on top of one of your markers!!

**Comment by epham36** [comment]
https://epham36.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@epham36 unable to see survey, please fix when you get a chance.

**Reply by albertkun** [reply]
@epham36 I see I have permission now! But, I think the pop-ups are broken! 🤔 
![image](https://user-images.githubusercontent.com/8574425/117251091-08247100-adf9-11eb-869d-24f791cee0f6.png)

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@skarakas1 Looks good! I added some info about my hometown!

**Comment by KenNL42** [comment]
https://kennl42.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@KenNL42 Looks good! I added a point, but remember one of the goals of the assignment was to use data fields, your survey and pop-up only has 1!

**Comment by vcta99** [comment]
https://vcta99.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@vcta99 Wow! Nice survey!!! I like how you added the counter at the bottom!!! I also added a point, see if you can find it. I put "Westwood, CA" and that's where it ended up.

**Comment by okcirE** [comment]
https://okcire.github.io/21S-ASIAAM-191A-Assignments/LAB_5/index.html

**Comment by rh5140** [comment]
https://rh5140.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Comment by jsxu315** [comment]
https://jsxu315.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Reply by albertkun** [reply]
@jsxu315 wow, nice random marker!!! but.. i hope this didn't keep you up!!!!

**Comment by annyrviloria** [comment]
https://annyrviloria.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html

**Comment by Wny-Duong** [comment]
https://wny-duong.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html
(Resubmitted because editing the survey to make it public screwed up sheet access :sweat: )

**Comment by lvalles10** [comment]
https://lvalles10.github.io/21S-ASIAAM-191A-Assignments/Week_05/

**Comment by ninalong** [comment]
https://ninalong.github.io/21S-ASIAAM-191A-Assignments/Week_05/index.html
